commons-numbers-complex-arrays/src/test/java/org/apache/commons/numbers/complex/arrays/ComplexListTest.java

```diff
@@ -349,10 +561,14 @@ private static <T> void assertListOperation(Function<List<Complex>, T> operation
         Assertions.assertEquals(l1, l2);
     }
 
-    private static <T> void assertListOperation(Function<List<Complex>, T> operation) {
+    private static <T> void assertListOperation1(Function<List<Complex>, T> operation) {
         assertListOperation(operation, new ArrayList<>(), ComplexList.interleaved());
     }
 
+    private static <T> void assertListOperation2(Function<List<Complex>, T> operation) {
```

Comment:
`assertListOperationNonInterleaved`

Looking at the use cases for this method, I think it could be removed if tests are broken down and parameterized.